feature-group-definitions/simd.yml

```
@@ -0,0 +1,6 @@
+name: SIMD
```

Even though it's not minimal, I think we should follow caniuse in name and ID here:

```suggestion
name: WebAssembly SIMD
```

There's no feature called just SIMD that to disambiguate from, but SIMD is a concept outside of web technologies, and I think "this browser supports SIMD" sounds a bit off, a bit like "this browser supports GPUs".

---

Thank you! I initially included WebAssembly but couldn't articulate _why_, so I assumed it was wrong. You explained it perfectly.